src/Tickets/Integrations/Plugins/Events_Pro/Duplicate_Post.php

```diff
@@ -74,14 +74,15 @@ public function duplicate_tickets_to_new_post( \WP_Post $new_post, \WP_Post $pos
 			return;
 		}
 
-		$provider = tribe( $tickets[0]->provider_class );
+		// You technically can have multiple providers if you have RSVP + a ticket provider.
+		foreach ( $tickets as $ticket ) {
 
-		// Check to see if we have a provider.
-		if ( empty( $provider ) || ! $provider instanceof Tribe__Tickets__Tickets ) {
-			return;
-		}
+			$provider = tribe( $ticket->provider_class );
+
+			if ( empty( $provider ) || ! $provider instanceof Tribe__Tickets__Tickets ) {
+				return;
```

Comment:
Should this be `continue` so it doesn't break out of the loop?
```suggestion
				continue;
```